### PR TITLE
New feature of daal fry done

### PR DESCRIPTION
**Pull request by prajapatihb**
Kindly review my code and approve the request.

### CODE REVIEW

File: main/daal-fry/recipe.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+1. Wash and soak lentils.
+2. Pressure cook lentils with turmeric and salt.
+3. In a pan, heat oil and add cumin seeds, garlic, and ginger.
+4. Sauté onions until golden brown.
+5. Add tomatoes and spices, and cook until oil separates.
+6. Add cooked lentils and water, and simmer until desired consistency.
```
Comment (harshadbprajapati) [inline main/daal-fry/recipe.txt]: Add the last step of garnishing.

File: test/daal-fry/recipeTest.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+1. Look: Check the consistency and color of the dal.
```
Comment (harshadbprajapati) [inline test/daal-fry/recipeTest.txt]: It is daal and not dal.

File: test/daal-fry/recipeTest.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+2. Taste: Taste the dal to ensure it's well-spiced and flavorful.
```
Comment (harshadbprajapati) [inline test/daal-fry/recipeTest.txt]: It is daal and not dal.

**Review by harshadbprajapati — CHANGES_REQUESTED** [review]
Kindly do the changes.

File: main/daal-fry/recipe.txt (hunk 1 above)
Comment (prajapatihb) [reply main/daal-fry/recipe.txt]: The last step of garnishing has been added.

File: test/daal-fry/recipeTest.txt (hunk 2 above)
Comment (prajapatihb) [reply test/daal-fry/recipeTest.txt]: The spelling dal -> daal has been replaced at both the places

File: test/daal-fry/recipeTest.txt (hunk 3 above)
Comment (prajapatihb) [reply test/daal-fry/recipeTest.txt]: The spelling dal -> daal has been replaced at both the places

**Review by harshadbprajapati — APPROVED** [review]
Good work. Your PR has been approved.

**Pull request merged**